### PR TITLE
Giphy give me the gifs you git

### PR DESCRIPTION
**Pull request by muz**
The most important PR ever

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "hubot-diagnostics": "0.0.1",
     "hubot-forecast": "^0.2.0",
     "hubot-github-compare": "^1.0.1",
+    "hubot-giphy-gifme": "^0.1.2",
     "hubot-google-images": "^0.1.4",
```
Comment (keynmol) [inline package.json:17]: This can go now, right?

**Comment by 36degrees** [comment]
MAKE IT RAAAAIN

File: package.json (hunk 1 above)
Comment (muz) [inline package.json:17]: neva4get

**Pull request merged**